doc/dox_comments/header_files/asn_public.h

```diff
@@ -1452,6 +1452,50 @@ int wc_EccPublicKeyDecode(const byte* input, word32* inOutIdx,
 int wc_EccPublicKeyToDer(ecc_key* key, byte* output,
                                          word32 inLen, int with_AlgCurve);
 
+/*!
+    \ingroup ASN
+
+    \brief This function converts the ECC public key to DER format. It
+    returns the size of buffer used. The public ECC key in DER format is stored
+    in output buffer. with_AlgCurve is a flag for when to include a header that
+    has the Algorithm and Curve information. The comp parameter
+
+    \return >0 Success, size of buffer used
+    \return BAD_FUNC_ARG Returned if output or key is null.
+    \return LENGTH_ONLY_E Error in getting ECC public key size.
+    \return BUFFER_E Returned when output buffer is too small.
+
+    \param key Pointer to ECC key
+    \param output Pointer to output buffer to write to.
+    \param inLen Size of buffer.
+    \param with_AlgCurve a flag for when to include a header that has the
+    Algorithm and Curve information.
+    \param comp Boolean value. If true the ECC public key will be written in
```

Comment:
I would prefer 1=true and 0=false instead of indicating "Boolean". This matches the example.